src/Raven.Server/Documents/Queries/Results/TimeSeriesRetriever.cs

```diff
@@ -726,10 +731,17 @@ private BlittableJsonReaderObject GetFinalResult(DynamicJsonArray array, long co
 
             if (addProjectionToResult)
             {
-                result[Constants.Documents.Metadata.Key] = new DynamicJsonValue
+                var metadata = new DynamicJsonValue
                 {
                     [Constants.Documents.Metadata.Projection] = true
                 };
+                var config = _context.DocumentDatabase.ServerStore.Cluster.ReadTimeSeriesConfiguration(_context.DocumentDatabase.Name);
```

Comment:
This opens an additional tx for server store for each query. Do we need that? Is this for Studio? If yes, IIRC we have that information during the query, so we can only do that when using Studio.